apollo-router/src/uplink/mod.rs

```diff
@@ -149,13 +152,20 @@ where
                             id,
                             response,
                             delay,
+                            ordering_id,
                         } => {
                             last_id = Some(id);
-                            if sender.send(Ok(response)).await.is_err() {
-                                break;
-                            }
-
                             interval = Duration::from_secs(delay);
+
+                            // If the ordering_id was less than the last then don't bother with this event as one of the uplink replicas is probably behind.
+                            if ordering_id > last_ordering_id {
+                                last_ordering_id = ordering_id;
+                                if sender.send(Ok(response)).await.is_err() {
+                                    break;
+                                }
+                            } else {
```

Comment:
let s only log if the ordering_id < last_ordering_id

Reply:
Fixed in [d8fa3d7](https://github.com/apollographql/router/pull/2803/commits/d8fa3d7c3d892546dc509d65192385b75e8f3d38)